pom.xml

```diff
@@ -835,15 +835,11 @@
               <artifactId>checkstyle</artifactId>
               <version>6.19</version>
             </dependency>
-            <dependency>
-              <groupId>org.apache.beam</groupId>
-              <artifactId>beam-sdks-java-build-tools</artifactId>
-              <version>${project.version}</version>
-            </dependency>
           </dependencies>
           <configuration>
-            <configLocation>beam/checkstyle.xml</configLocation>
-            <suppressionsLocation>beam/suppressions.xml</suppressionsLocation>
+            <!-- beam.rootdir set by scjijava:set-rootdir -->
+            <configLocation>${beam.rootdir}/sdks/java/build-tools/src/main/resources/beam/checkstyle.xml</configLocation>
```

Comment:
So, my total proposal is indeed to flush the module, I just wanted to give @dhalperi  a look before I did that part.